framework/src/mfem/bcs/MFEMVectorFunctorBoundaryIntegratedBC.C

```diff
@@ -0,0 +1,42 @@
+#ifdef MFEM_ENABLED
+
+#include "MFEMVectorFunctorBoundaryIntegratedBC.h"
+
+registerMooseObject("MooseApp", MFEMVectorFunctorBoundaryIntegratedBC);
+
+InputParameters
+MFEMVectorFunctorBoundaryIntegratedBC::validParams()
+{
+  InputParameters params = MFEMIntegratedBC::validParams();
+  params.addRequiredParam<MFEMVectorCoefficientName>(
+      "vector_coefficient",
+      "Vector coefficient used in the boundary integrator. A coefficient can be any of the "
+      "following: a "
+      "variable, an MFEM material property, a function, or a post-processor.");
+  return params;
+}
+
+// TODO: Currently assumes the vector function coefficient is 3D
```

Comment:
I should have deleted this in a previous PR. Tidying up now that I'm thinking of it.
```suggestion
```